packages/millicast-sdk/tests/features/step-definitions/LoggerDiagnose.steps.js

```diff
@@ -0,0 +1,151 @@
+import { loadFeature, defineFeature } from 'jest-cucumber'
+import Logger from '../../../src/Logger'
+import View from '../../../src/View'
+import MockRTCPeerConnection, { rawStats } from './__mocks__/MockRTCPeerConnection'
+import './__mocks__/MockMediaStream'
+import './__mocks__/MockBrowser'
+import Publish from '../../../src/Publish'
+import Diagnostics from '../../../src/utils/Diagnostics'
+
+const feature = loadFeature('../LoggerDiagnose.feature', { loadRelativePath: true, errors: true })
+
+jest.mock('../../../src/Signaling')
+
+const expectedObject = {
+  accountId: expect.any(String),
+  streamName: expect.any(String),
+  subscriberId: expect.any(String),
+  connection: expect.any(String),
+  version: expect.any(String),
```

Comment:
maybe we could use fixed/constant version and streamName for better verification.

Reply:
streamName cannot be fixed as it does not return the one passed, because the method that saves it is mocked